portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -38,6 +38,18 @@
   private static final Double LNG_WEST_LIMIT = -106.441602;
   private static final Double LNG_EAST_LIMIT = -106.424213;
 
+  class Result {
```

Comment:
needs to be `static class` or it will be an instance class, which is expensive. Also move to the bottom of the file with the enums.

Reply:
Should the static class and the enums be inside MarkerServlet public class?

Reply:
There's no clear guidance. It's only used in the servlet so it's fine to leave it here. If it's used in multiple places, we would need to move it into a separate file.